### PR TITLE
Add DOMPurify and sanitize puzzle and lab descriptions

### LINKED ISSUE #48: Update formatting in puzzle descriptions
Puzzle descriptions - would be nice if italics/etc. showed up analogously to the website version (example - see last line in attached). Tested on 1.4.1 on iPhone 8.

pic1

### PR DESCRIPTION
**Pull request by tkaragianes**
Closes #48. User provided descriptions of puzzles and labs often include HMTL styling, links, images, etc. Now this HTML will properly render, after being sanitized by DOMPurify to protect against XSS attacks.

### CODE REVIEW

File: src/views/LabView.vue [hunk 1]
```diff
@@ -187,11 +188,11 @@ export default Vue.extend({
             return this.$store.state.current_lab.lab.title;
         },
         descriptiontoShow(): string{
-            // strip all html, only take the first MAX_CHARS characters and add ending ... tag
-            return this.$store.state.current_lab.lab.body.replace(/<[^>]+>/ig,"").substr(0, MAX_CHARS) + ". . .";
+            // Limit to MAX_CHARS and add ellipsis
+            return DOMPurify.sanitize(`${this.$store.state.current_lab.lab.body.substr(0, MAX_CHARS)}...`)
```
Comment (luxaritas) [inline src/views/LabView.vue:192]: This could cause broken HTML. Can we rely on CSS for elipsis?

**Review by luxaritas — COMMENTED** [review]
FYI it looks like you accidentally reverted the EternaJS submodule

File: src/views/LabView.vue (hunk 1 above)
Comment (tkaragianes) [reply src/views/LabView.vue:192]: Happily. Just matching what was there before.

**Comment by tkaragianes** [comment]
Good catch on the Eterna revert.

**Pull request closed without merging**